example-prjs/simple-demo/firmware/parameters.h

```diff
@@ -0,0 +1,46 @@
+#ifndef PARAMETERS_H_
+#define PARAMETERS_H_
+
+#include <complex>
+#include "ap_int.h"
+#include "ap_fixed.h"
+#include "nnet_layer.h"
+
+//hls-fpga-machine-learning insert numbers
+#define N_INPUTS 10
+#define N_LAYER_1 32
+#define N_OUTPUTS 1
+
+typedef ap_fixed<32,10> accum_t;
```

Comment:
So these 7 lines go away and the default values are specified in the YAML?

Reply:
I havent gotten that far yet! If anything, I think the general list of typedefs would go away, to be replaced by specific typedefs inside each struct.